sensu-plugins-slack.gemspec

```diff
@@ -38,7 +38,7 @@ Gem::Specification.new do |s|
   s.add_development_dependency 'pry',                       '~> 0.10'
   s.add_development_dependency 'rake',                      '~> 10.0'
   s.add_development_dependency 'redcarpet',                 '~> 3.2'
-  s.add_development_dependency 'rubocop',                   '~> 0.40.0'
   s.add_development_dependency 'rspec',                     '~> 3.1'
+  s.add_development_dependency 'rubocop',                   '~> 0.52.1'
```

Comment:
This needs to be called out in the changelog under `### Breaking Changes`  so that we inform users on the impact and make it more obvious to reviewers and maintainers so it can be evaluated and versioned appropriately.